src/ImageSharp/Formats/Jpeg/MetadataExtensions.cs

```diff
@@ -17,4 +18,37 @@ public static partial class MetadataExtensions
     /// <param name="metadata">The metadata this method extends.</param>
     /// <returns>The <see cref="JpegMetadata"/>.</returns>
     public static JpegMetadata GetJpegMetadata(this ImageMetadata metadata) => metadata.GetFormatMetadata(JpegFormat.Instance);
+
+    /// <summary>
+    /// Sets the comment in <see cref="JpegMetadata"/>
+    /// </summary>
+    /// <param name="metadata">The metadata this method extends.</param>
+    /// <param name="index">The index of comment to be inserted to.</param>
+    /// <param name="comment">The comment string.</param>
+    public static void SetComment(this JpegMetadata metadata, int index, string comment)
```

Comment:
Do we really need such methods if comments are exposed as `IList`?
`IList` has all this methods so users can do whatever they want.

Reply:
My thoughts exactly.